### PR TITLE
add check to show list if media is saved on media list page

### PR DESCRIPTION
**Pull request by rnagulapalle**
[[Media][Bug] Javascript Error on Empty Media Library](https://app.asana.com/0/596117517607240/770981223101979)

### CODE REVIEW

File: components/mediaLibrary.html [hunk 1]
```diff
@@ -78,7 +78,7 @@
   </div>
 
   <!-- Media Library -->
-  <div ng-if="$ctrl.showLibrary" class="media-library-content">
+  <div ng-if="$ctrl.showLibrary" class="media-library-content" ng-if="collection.models.length > 0 ">
```
Comment (alexgurrola) [inline components/mediaLibrary.html]: There should only be 1 `ng-if` per tag.

**Comment by alexgurrola** [comment]
This now has more merge commits than before, which all need to be rebased.

**Comment by alexgurrola** [comment]
This also doesn't pass Standard JS styling anymore.

File: components/mediaLibrary.html (hunk 1 above)
Comment (rnagulapalle) [reply components/mediaLibrary.html]: am working on this today and will update you..

File: components/userAuthentication.js [hunk 2]
```diff
@@ -133,20 +133,28 @@
         $ctrl.user.set('token', utility.getUrlParams().token)
         $ctrl.user.save().then(function () {
           $ctrl.loading = false
-          $ctrl.message = _.first($ctrl.user.meta.get('status')).message
-          $ctrl.enabledVerificationForm = false
-          $ctrl.isRequestSuccess = !$ctrl.user.error
-          if (!$ctrl.user.error) {
+          if (utility.getStatus(response).code === utility.RESPONSE_CODE.verify) {
```
Comment (alexgurrola) [inline components/userAuthentication.js]: This won't work with any of our data structure and needs to be reverted to:

```js
$ctrl.message = _.first($ctrl.user.meta.get('status')).message
$ctrl.enabledVerificationForm = false
$ctrl.isRequestSuccess = !$ctrl.user.error
if (!$ctrl.user.error) {
```

**Comment by alexgurrola** [comment]
This still needs a rebase.

**Comment by rnagulapalle** [comment]
closing because rebase messed up changes

**Pull request closed without merging**